packages/block-library/src/latest-posts/index.php

```diff
@@ -25,24 +25,22 @@ function render_block_core_latest_posts( $attributes ) {
 
 	$list_items_markup = '';
 
-	foreach ( $recent_posts as $post ) {
-		$post_id = $post['ID'];
-
-		$title = get_the_title( $post_id );
+	foreach ( $recent_posts_query->posts as $post ) {
+		$title = get_the_title( $post->ID );
```

Comment:
You can just pass `$post` instead of `$post->ID`. Makes the code a bit easier to read/follow.

Reply:
fixed, Thanks